WordPressKit/BlogServiceRemote.h

```diff
@@ -22,6 +22,17 @@ typedef void (^SuccessHandler)(void);
 - (void)getAuthorsWithSuccess:(UsersHandler)success
                       failure:(void (^)(NSError *error))failure;
 
+/**
+ Synchronizes all blog's authors.
+
+ @param number Limit the total number of users returned.
+ @param success The block that will be executed on success.  Can be nil.
+ @param failure The block that will be executed on failure.  Can be nil.
+ */
+- (void)getAllAuthorsWithNumber:(NSNumber *)number
```

Comment:
Maybe its just me, but is it a little weird for a method named "get all authors" to then set a limit for the number returned?   Maybe just `getAuthorsWithNumber` ?